extras/ams_wrapper/src/api/ovms_connector.py

```diff
@@ -25,12 +31,35 @@ def __init__(self, ovms_port, ovms_model_info):
         self.model_version = ovms_model_info['model_version']
         self.input_name = ovms_model_info['input_name']
         self.input_shape = ovms_model_info['input_shape']
+        self.output_name = ovms_model_info["output_name"] 
 
         channel = grpc.insecure_channel("{}:{}".format("127.0.0.1", self.ovms_port))
         self.stub = prediction_service_pb2_grpc.PredictionServiceStub(channel)
-    
+
     def send(self, inference_input):
         # TODO: prepare request and handle response
         request = predict_pb2.PredictRequest()
-        result = self.stub.Predict(request, 10.0)
-        return 
+        request.model_spec.name = self.model_name
+        try:
+            tensor_proto = make_tensor_proto(inference_input, shape=(inference_input.shape))
+        except TypeError as e:
+            raise TypeError("Unsupported data type") from e
+        except ValueError as e:
+            raise ValueError("Invalid arguments") from e
+        request.inputs[self.input_name].CopyFrom(tensor_proto)
+        try:
+            result = self.stub.Predict(request, 10.0)
+            output = make_ndarray(result.outputs[self.output_name])
+            numpy_array = np.array(output)
+        except grpc.RpcError as e:
+            if e.code() == grpc.StatusCode.INVALID_ARGUMENT:
+                raise ValueError("Invalid argument: {}".format(e.details())) from e
+            elif e.code() == grpc.StatusCode.NOT_FOUND:
+                raise ModelNotFoundError("Requested model not found") from e
+            else:
```

Comment:
I think we should explicitly handle errors like SOCKET_CLOSED for situations when OVMS is not accessible by the AMS.